Source/items.cpp

```diff
@@ -4558,8 +4562,6 @@ BOOL PremiumItemOk(int i)
 		rv = FALSE;
 	if (AllItemsList[i].itype == ITYPE_GOLD)
 		rv = FALSE;
-	if (AllItemsList[i].itype == ITYPE_FOOD)

```

Comment:
Would be nice to have this `ITYPE_FOOD` in its own PR.

Reply:
I split it into its own commit, best I can do here.